unit-tests/run-unit-tests.py

```diff
@@ -468,6 +477,9 @@ def get_tests():
 
         yield PyTest(testname, py_test)
 
+def prioritize_tests( tests ):
+    for test in sorted(tests, key= lambda t: t.config.priority):

```

Comment:
Hmm... can't we just `return sorted( ... )`?

Reply:
You told me to use yield... I can change to return the list